test/snapshots/index.js.md

```diff
@@ -4,6 +4,15 @@ The actual snapshot is saved in `index.js.snap`.
 
 Generated by [AVA](https://avajs.dev).
 
+## accessing the broker using the createChecker factory
+
+> Snapshot 1
+
+    [
+      503,
```

Comment:
OK `503`, how does it guarantee that this 503 is due to the `next(b.healthcheck.port)`?

Returning the error message in the response would guarantee that.

Reply:
Fixed in 09f3f50